client/src/Client.cs

```diff
@@ -33,34 +30,35 @@ static int Main(string[] arguments)
         {
             Action printUsage = () =>
             {
-                Console.WriteLine("Usage: Client <hostname> <port> <client_id>");
-                Console.WriteLine("Connects to a demo deployment.");
-                Console.WriteLine("    <hostname>      - hostname of the receptionist to connect to.");
-                Console.WriteLine("    <port>          - port to use.");
-                Console.WriteLine("    <client_id>     - name of the client.");
-                Console.WriteLine("Alternatively: Client <snapshotfile> will generate a snapshot and exit.");
+                Console.WriteLine("Usage 1: Client local <hostname> <port> <client id>");
+                Console.WriteLine("Connects to a local deployment.");
+                Console.WriteLine("");
+                Console.WriteLine("Usage 2: Client cloud <hostname> <player identity token> <login token>");
+                Console.WriteLine("Connects to a cloud deployment.");
+                Console.WriteLine("");
+                Console.WriteLine("Usage 3: Client snapshot <snapshot file>");
+                Console.WriteLine("Generate a snapshot and exit.");
```

Comment:
Minor thing (consistency) - should this be "Generates a snapshot and exits" to match the verb forms used in 1 and 2? (Or alternatively we could change the others to "Connect".)

Reply:
Good point. Changed.